Examples/Swift/HeatmapExample.swift

```diff
@@ -26,12 +25,12 @@ class HeatmapExample: UIViewController, MGLMapViewDelegate {
         let heatmapLayer = MGLHeatmapStyleLayer(identifier: "earthquakes", source: source)
         
         // Adjust the color of the heatmap based on the point density.
-        let colorDictionary : [NSNumber : UIColor] = [
-                                0.0 :  .clear,
-                               0.01 : .white,
-                               0.15 : UIColor(red:0.19, green:0.30, blue:0.80, alpha:1.0),
-                               0.5 : UIColor(red:0.73, green:0.23, blue:0.25, alpha:1.0),
-                               1 : .yellow
+        let colorDictionary: [NSNumber: UIColor] = [
+                                0.0: .clear,
+                               0.01: .white,
+                               0.15: UIColor(red: 0.19, green: 0.30, blue: 0.80, alpha: 1.0),
+                               0.5: UIColor(red: 0.73, green: 0.23, blue: 0.25, alpha: 1.0),
+                               1: .yellow
```

Comment:
I can change this back and align by colons instead of by the first character. From what I could tell, there doesn't seem to be a rule for aligning colons in collection types.